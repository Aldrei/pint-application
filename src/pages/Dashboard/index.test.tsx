import React from 'react';

import { render } from '@testing-library/react';

jest.mock('../../hooks/useReducerSelector', () => ({
  useAppSelectorBlaBlaBal: jest.fn(() => ({
    status: 'idle',
    accessToken: 'abc123'
  }))
}));

import DashboardPage from './index';

describe('Login page', () => {
  it('Should be render correctly', () => {
    const nodeEl = render(<DashboardPage />);

    expect(nodeEl.baseElement).toHaveTextContent('Você está logado!');

    expect(nodeEl.baseElement).toMatchSnapshot();
  });
});
