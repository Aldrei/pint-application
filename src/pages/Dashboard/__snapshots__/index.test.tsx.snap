// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page Should be render correctly 1`] = `
<body>
  <div>
    <div
      class="sc-bczRLJ jzOFOE"
      data-testid="dashboard-container"
    >
      <div
        class="cardComp css-p1o44d"
      >
        Você está logado!
      </div>
    </div>
  </div>
</body>
`;
